### PR TITLE
Add note for oidc dynamic query params

### PR DESCRIPTION
**Pull request by sherenewso2**
## Purpose
Resolves https://github.com/wso2/product-is/issues/10536

### CODE REVIEW

File: en/docs/learn/configuring-oauth2-openid-connect.md [hunk 1]
```diff
@@ -56,7 +56,8 @@ the users with an authorization server-based authentication.
     
         If you want to change the format in which the **Client Secret** is
         stored, open the `	<IS_HOME>/repository/conf/deployment.toml	` file and add the following configuration.
-        ```xml
+
+        ```toml
         [oauth]
 		hash_tokens_and_secrets = true
```
Comment (gomathyk) [inline en/docs/learn/configuring-oauth2-openid-connect.md:62]: Make sure this is aligned. Doesn't seem proper in the git view.

File: en/docs/learn/configuring-oauth2-openid-connect.md [hunk 2]
```diff
@@ -70,20 +71,73 @@ the users with an authorization server-based authentication.
...
+    <tr>
+        <th>Field </th>
+        <th>Description</th>
+        <th>Sample Value</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>Enable OAuth2/OpenIDConnect</td>
+        <td>Selecting this option enables OAuth2/OpenID Connect to be used as an authenticator for users provisioned to the Identity Server.</td>
```
Comment (gomathyk) [inline en/docs/learn/configuring-oauth2-openid-connect.md:85]: WSO2 Identity Server. Please change everywhere.

File: en/docs/learn/configuring-oauth2-openid-connect.md [hunk 3]
```diff
@@ -70,20 +71,73 @@ the users with an authorization server-based authentication.
...
+        <td>Selected</td>
+    </tr>
+    <tr>
+        <td>Default</td>
+        <td>Selecting the Default check box signifies that the OAuth2/OpenID Connect credentials are the main/default form of authentication. <br>This removes the selection made for any other Default checkboxes for other authenticators.</td>
```
Comment (gomathyk) [inline en/docs/learn/configuring-oauth2-openid-connect.md:90]: Make 'D' simple in Default (both places) or boldify.

File: en/docs/learn/configuring-oauth2-openid-connect.md [hunk 4]
```diff
@@ -70,20 +71,73 @@ the users with an authorization server-based authentication.
...
+        <td><code>https://localhost:9443/oauth2/token/</code></td>
+    </tr>
+    <tr>
+        <td>Client Id</td>
+        <td>Client ID of the application you registered in the IDP for Identity Server.</td>
+        <td><code>1421263438188909</code></td>
+    </tr>
+    <tr>
+        <td>Client Secret</td>
+        <td>Client Secret of the application you registered in the IDP for Identity server. Click the Show button to view the value you enter.</td>
```
Comment (gomathyk) [inline en/docs/learn/configuring-oauth2-openid-connect.md:110]: Shall we use bold on "show" here?

File: en/docs/learn/configuring-oauth2-openid-connect.md [hunk 5]
```diff
@@ -70,20 +71,73 @@ the users with an authorization server-based authentication.
...
+        <td><code>12ffb4dfb2fed67a00846b42126991f8</code></td>
+    </tr>
+    <tr>
+        <td>Callback URL</td>
+        <td>This is the URL to which the browser should be redirected after the authentication is successful. It should be the <code>commonauth</code> endpoint of Identity server</td>
+        <td><code>https://localhost:9443/commonauth</code></td>
+    </tr>
+    <tr>
+        <td>OpenID Connect User ID Location</td>
+        <td>Select whether the User ID is found in the 'sub' attribute that is sent with the OpenID Connect request or if it is found among claims.</td>
```
Comment (gomathyk) [inline en/docs/learn/configuring-oauth2-openid-connect.md:120]: Selects?

**Review by gomathyk — CHANGES_REQUESTED** [review]

File: en/docs/learn/configuring-oauth2-openid-connect.md (hunk 1 above)
Comment (sherenewso2) [reply en/docs/learn/configuring-oauth2-openid-connect.md:62]: this is fine in the editor

File: en/docs/learn/configuring-oauth2-openid-connect.md (hunk 5 above)
Comment (sherenewso2) [reply en/docs/learn/configuring-oauth2-openid-connect.md]: Here we are instructing the user to select an option, so using 'select' is appropriate right?

**Review by gomathyk — APPROVED** [review]

**Pull request merged**